src/screens/Calendar/MeetingModal/DisplayMeetingModal/DisplayAttendees.tsx

```diff
@@ -56,6 +64,9 @@ const InvitedMember = ({ member }) => {
         <p onClick={() => navigate(`/users/${member.userId}`)}>{memberName}</p>
         <p className='role'>{member.role}</p>
       </div>
+      {member.organizer === true ? (
+        <p className='organizer'>Organizer</p>
+      ) : null}
```

Comment:
```suggestion
      {member.organizer && (
        <p className='organizer'>Organizer</p>
      )}
```

Can this be condensed to use the guard operator `&&` instead of a null ternary?